examples/container/serverless.yml

```diff
@@ -0,0 +1,31 @@
+service:
+  name: scaleway-container
+
+provider:
+  name: scaleway
+  # Global Environment variables - used in every functions
+  env:
+    test: test
+  # the path to the credentials file needs to be absolute
+  scwToken: <scw-token>
+  scwOrganization: <scw-organization-id>
+
+plugins:
+  - serverless-scaleway-functions
+
+package:
+  exclude:
+    - node_modules/**
+    - .gitignore
+    - .git/**
+
+custom:
+  first:
```

Comment:
Your containers should not be defined strictly under `custom` parameter, as the custom parameter is used by many other serverless plugins.

You should instead add your container definitions under a variable `scalewayContainers` for example:
```yaml
custom:
  scalewayContainers:
    first:
      directory: ......
```